src/modules/terminalfont.c

```diff
@@ -69,6 +67,51 @@ static void printXFCE4Terminal(FFinstance* instance)
     printTerminalFont(instance, font);
 }
 
+static void printTilixTerminal(FFinstance* instance)
+{
+    static const char* fontName = NULL;
+    static const char* defaultProfile = NULL;
+    static FFstrbuf key;
+    static FFvariant res;
```

Comment:
Why is `static` needed here? They are never needed outside of the function. This way it adds unnecessary symbols.